.github/workflows/ci.yaml

```diff
@@ -0,0 +1,54 @@
+name: CI
+on: [push]
+
+jobs:
+  lint:
+    name: Lint
+
+    runs-on: ubuntu-22.04
+
+    strategy:
+      matrix:
+        module:
+          - slog
```

Comment:
Did not find a nice way to avoid duplicating this module list, there is no proper variables in GHA, only env vars (which are always just a plain string).